tests/test_ega.py

```diff
@@ -15,7 +15,7 @@
 # IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
-
+from click.testing import CliRunner
 from argparse import Namespace
```

Comment:
This and the dependency on argparse still needed? If not, lets remove them both. Thanks.